db/src/main/resources/db/migration/V1_047__remove_varchar_size_contraints.sql

```diff
@@ -0,0 +1,26 @@
+-- Remove size constraints from all varchar columns where they are specified
+ALTER TABLE address ALTER COLUMN address_line_1 TYPE varchar;
+ALTER TABLE address ALTER COLUMN address_line_2 TYPE varchar;
+ALTER TABLE address ALTER COLUMN town_or_city TYPE varchar;
+ALTER TABLE address ALTER COLUMN postcode TYPE varchar;
+ALTER TABLE address ALTER COLUMN county TYPE varchar;
+ALTER TABLE claim ALTER COLUMN dwp_household_identifier TYPE varchar;
```

Comment:
Can we put a blank line between statements for different tables?

Reply:
Done.